### ISSUE TITLE
is the code correct?

### ISSUE THREAD

**Issue by tankonf** [omkar98/InfyTQ-Answers#18]
in DBMS/Day-3/Assignment-17.txt is the code correct and working?

**Comment by omkar98** [comment]
@tankonf Yeah. I think you need to do some minor tweaks in the editor to get this working.
